services/tenant-ui/frontend/src/components/issuance/Schemas.vue

```diff
@@ -23,12 +23,22 @@
           </span>
         </div>
       </template>
+      <Column :sortable="false" header="Actions">
+      </Column>
       <Column field="name" header="Schema" filter-field="name" />
       <Column field="version" header="Version" />
       <Column field="status" header="Status" />
       <Column field="state" header="State" />
       <Column field="attributes" header="Attributes" />
       <Column field="schema_id" header="ID" />
+      <Column field="credential_templates" header="Credential Template">
+        <template #body="{data}">
+          <CreateCredentialTemplate :schema-template-id="data.schema_template_id" v-if="data.credential_templates.length == 0"/>
```

Comment:
I think can leave off the ` == 0` for the boolean expression (`v-if="data.credential_templates.length")